File: app/src/main/java/com/surya/shopcart/thankyou/ThankyouActivity.java
package com.surya.shopcart.thankyou;

import android.content.Intent;
import android.os.Bundle;
import android.view.MenuItem;
import android.widget.Button;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

import com.google.android.gms.tasks.OnSuccessListener;
import com.google.firebase.auth.FirebaseAuth;
import com.google.firebase.auth.FirebaseUser;
import com.surya.shopcart.ProductHomePageActivity;
import com.surya.shopcart.R;
import com.surya.shopcart.confirmorder.ConfirmOrderActivity;
import com.surya.shopcart.utils.Utils;

public class ThankyouActivity extends AppCompatActivity {


    String userId;
    public void openCartPage(MenuItem item) {
        Utils.handleMenuCLick(this, item);
    }
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_thankyou);

        // Home navigation icon
        Utils.addHomeIconNavigation(this, findViewById(R.id.topAppBar));


        Button homePage = findViewById(R.id.homePage);

        FirebaseAuth mAuth = FirebaseAuth.getInstance();
        FirebaseUser currentUser = mAuth.getCurrentUser();

        if (currentUser != null) {
            userId = currentUser.getUid();
        }

        homePage.setOnClickListener(view -> {
            //Proceed to home page
            Intent productPage = new Intent(getApplicationContext(), ProductHomePageActivity.class);
            startActivity(productPage);
            finish();
        });

    }
}

File: app/src/main/java/com/surya/shopngo/thankyou/ThankyouActivity.kt
package com.surya.shopngo.thankyou

import android.content.Intent
import android.os.Bundle
import android.view.MenuItem
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import com.google.firebase.auth.FirebaseAuth
import com.surya.shopngo.ProductHomePageActivity
import com.surya.shopngo.R
import com.surya.shopngo.utils.Utils

class ThankyouActivity : AppCompatActivity() {
    var userId: String? = null
    fun openCartPage(item: MenuItem?) {
        Utils.handleMenuCLick(this, item)
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_thankyou)

        // Home navigation icon
        Utils.addHomeIconNavigation(this, findViewById(R.id.topAppBar))
        val homePage = findViewById<Button>(R.id.homePage)
        val mAuth = FirebaseAuth.getInstance()
        val currentUser = mAuth.currentUser
        if (currentUser != null) {
            userId = currentUser.uid
        }
        homePage.setOnClickListener { view: View? ->
            //Proceed to home page
            val productPage = Intent(applicationContext, ProductHomePageActivity::class.java)
            startActivity(productPage)
            finish()
        }
    }
}